src/webgpu/api/operation/queue/writeBuffer.spec.ts

```diff
@@ -8,3 +8,89 @@ import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
```

Comment:
Validation tests should be in `api/validation` instead of `api/operation`. Looks like this test only tests validation and exceptions, doesn't actually test uploaded results for valid cases, so it can just move there wholesale. When you do so, take a look at the file description that's there and update it so it's clear what is still TODO.